packages/main/src/DatePicker.js

```diff
@@ -472,10 +471,6 @@ class DatePicker extends DateComponentBase {
 			return;
 		}
 
-		if (isEnter(event)) {
-			this._updateValueAndFireEvents(event.target.value, true, ["change", "value-changed"]);
```

Comment:
I see that this has been added on purpose by team Rila with this PR https://github.com/SAP/ui5-webcomponents/pull/4129 Could you please check with Team Rila about this?

Reply:
We have added this due to some tests failings.
If you think you are OK after removing it - we are fine.

Angular's two way binding is dependent on `value-changed` and I would suggest you to recheck that scenario.

Reply:
@unazko as a follow up on the Martin's remark, you just need to ensure that on Enter, now that you remove is handling, the "change", and "value-changed" events are still fired. If fired we are fine with the Angular use case.

Reply:
I haven't tested this scenario with an Angular example, but we should be fine, as in the ui5-date-picker code <code>change</code> event and </code>value-changed</code> are always fired simultaneously via the <code>_updateValueAndFireEvents</code> function.